src/Workspaces/Core/Portable/Execution/PinnedRemotableDataScope.cs

```diff
@@ -28,12 +28,20 @@ internal sealed class PinnedSolutionInfo
         /// can benefit other requests or not
         /// </summary>
         public readonly bool FromPrimaryBranch;
+
+        /// <summary>
+        /// This indicates a version of this solution. remote host engine uses this version
+        /// to decide whether caching this solution will benefit other requests or not
+        /// </summary>
+        public readonly int WorkspaceVersion;
```

Comment:
Comment states this is a version of this solution, but the field name is Workspace version.
We should either rename this field or change the comment. Should the field name be changed to something like `LatestSolutionVersion` or `LastSolutionVersion`?

Reply:
Ah, seems like Solution already has a field called [WorkspaceVersion](http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Workspace/Solution/Solution.cs,42). Probably just adjust the comment then to mention it is solution's workspace version?

Reply:
sure